### PR TITLE
Update e2e README to show new organised fields

### PR DESCRIPTION
**Pull request by jackdelahunt**
## Description
Updates the e2e README to reflect changes in the structure for the `config.json` in the e2e tests

## How Has This Been Tested?
- N/A

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: test/e2e-tests/README.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ The structure of the `config.json` is in four sections, the top level fields, `g
 - `image_registry_username` - quay.io username
 - `image_registry_password` - quay.io password
 - `target_image_tags` - JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
+- `git_container_file_repo` - Git repo containing the container file
+- `git_container_file_revision` - Git branch in the container file repo
+- `container_relative_path` - Relative path from the root of the cotainer file repo to where the container file is
```
Comment (MarianMacik) [inline test/e2e-tests/README.md:26]:
```suggestion
- `container_relative_path` - Relative path from the root of the container file repo to where the container file is
```

**Review by MarianMacik — APPROVED** [review]
Looks good, just one typo.

File: test/e2e-tests/README.md (hunk 1 above)
Comment (jackdelahunt) [reply test/e2e-tests/README.md:26]: I misspell at least once a pr... thank you

**Review by biswassri — APPROVED** [review]
/lgtm

File: test/e2e-tests/README.md (hunk 1 above)
Comment (MarianMacik) [reply test/e2e-tests/README.md]: @jackdelahunt no worries, with this amount of text it happens.

**Pull request merged**